app/javascript/components/Results/RssNews/RssNews.tsx

```diff
@@ -1,62 +1,61 @@
 import React from 'react';
+import Moment from 'react-moment';
 import { GridContainer, Grid } from '@trussworks/react-uswds';
+interface RssNewsProps {
+  recommendedBy: string;
+  news?: {
+    title: string;
+    link: string;
+    description: string;
+    publishedAt: string;
+  }[];
+  parse(html: string): string | JSX.Element | JSX.Element[]; // eslint-disable-line no-undef
```

Comment:
What do you think about importing `parse` in this file rather than passing it as a parameter?
```js
import parse from 'html-react-parser';
```

Reply:
Why avoid repeating imports? That's how be pass functionality and methods around, e.g. we import the method `useContext` (and the whole React object) every time we need it, rather than do it once and pass it around.

```js
import React, { useContext } from 'react';
```

What's the difference with the `parse` method?

Reply:
RssNews component has been updated.